scripts/create_patches_files.py

```diff
@@ -53,7 +53,7 @@ def parse_arguments():
     )
     return parser.parse_args()
 
-def parse_patches(patches: List[Dict[str, Any]]):
+def parse_patches(patches: List[Dict[str, Any]], patch_id: None | str = None):
```

Comment:
Not sure if this would have the same problem as https://github.com/patrick-rivos/riscv-gnu-toolchain/commit/9fa5711c3a87b88c3d6a5f3754f90244abcf928d

Reply:
great catch, fixed now